src/config/errorConfig.js

```diff
@@ -1,20 +1,16 @@
-/* eslint-disable no-unused-vars */
+/* eslint no-multi-spaces: ["warn", { exceptions: { "VariableDeclarator": true } }] */
+
+export const ERROR_NO_ADDRESS            = 3212;
+export const ERROR_NO_CC                 = 3224;
+export const ADDRESS_ERROR               = 3505;
```

Comment:
Waarom zou je alleen bij deze een andere conventie aanhouden. Want nu lijkt het een beetje op de error van ERROR_NO_ADDRESS => 3212

Reply:
hehe, helaas heb je gelijk, ik heb ze allemaal rechtgetrokken met de naam in de lijst van de api